doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -232,6 +232,8 @@ nRF Desktop
   * The :ref:`CONFIG_DESKTOP_BLE_DONGLE_PEER_ID_INFO <config_desktop_app_options>` configuration option.
     It can be used to indicate the dongle peer identity with a dedicated event.
   * Updated the :ref:`nrf_desktop_fast_pair_app` to remove the Fast Pair advertising payload for the dongle peer.
+  * Removed Fast Pair discoverable advertising payload on Resolvable Private Address (RPA) rotation during discoverable advertising session.
```

Comment:
We need to have a new section for removed and move this sentence and one mentioned in line no:245 underneath the Removed header.